s2wav/s2let_math.py

```diff
@@ -0,0 +1,83 @@
+"""Compute basic math tiling. """
+import numpy as np
+
+
+def f_s2dw(k, B):
```

Comment:
Let's include types in method arguments (see `s2fft` for example).

Reply:
Good point, so something like [this](https://github.com/astro-informatics/s2fft/blob/8ef91ce513c774233326695e3dfa5553a807ae01/s2fft/wigner/turok.py#L43) line @alicjapolanska (of course adjusted for your functions _etc_.)